ciphey/__main__.py

```diff
@@ -1,4 +1,42 @@
-from .ciphey import main
+#! /usr/bin/env python3
+
+"""
+Ciphey: https://github.com/ciphey/ciphey
+"""
+
+import sys
+import platform
 
 if __name__ == "__main__":
-    main()
+    major = sys.version_info[0]
+    minor = sys.version_info[1]
+
+    python_version = (
+        str(sys.version_info[0])
+        + "."
+        + str(sys.version_info[1])
+        + "."
+        + str(sys.version_info[2])
+    )
+
+    if major != 3 or (major == 3 and minor < 6):
```

Comment:
I think you should change this to `if major < 3 or minor < 6:` because we don't support Python 2 or any version lower than Python 3.6 so it can be simplified.